### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by jwendell**
- CNI
- API
- Common files
- Release builder

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -946,8 +947,8 @@ honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod h1:rf3lG4BRIbNafJWh
 honnef.co/go/tools v0.0.0-20190418001031-e561f6794a2a/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 istio.io/api v0.0.0-20190515205759-982e5c3888c6/go.mod h1:hhLFQmpHia8zgaM37vb2ml9iS5NfNfqZGRt1pS9aVEo=
-istio.io/api v0.0.0-20191015210738-bfa91e88abf1 h1:pkQv7+x97NKR70CODSOZBt4XKvb2HJlOcHcNkju7gv8=
-istio.io/api v0.0.0-20191015210738-bfa91e88abf1/go.mod h1:UP3a6saAz6eoOaXQqzy49/ygIjzUcEGZ0QTFpcfmSEA=
+istio.io/api v0.0.0-20191106222414-63a2b9325b00 h1:s0dOrgpIfQ2jd568vgfKe6MmSW3yJv+dIwC1OfZ3/EM=
+istio.io/api v0.0.0-20191106222414-63a2b9325b00/go.mod h1:+cyHH83OwC0rFpwk8eXctzPNpiCAbB+r6kmMiAxxBHw=
```
Comment (howardjohn) [inline go.sum]: We should apply the same changes here: https://github.com/istio/istio/pull/18716/files

Comment (jwendell) [reply go.sum]: It's backported already.

**Pull request merged**